src/sagemaker_training/process.py

```diff
@@ -40,6 +40,29 @@
 _DEFAULT_BUF_SIZE = 1024 * 64
 
 
+def get_debugger_exception_classes():
+    """Set exception classes"""
+    exception_classes = []
+    if os.environ.get("USE_SMDEBUG") == "0":
+        logger.info(
+            "sagemaker debugger is not enabled, using exception types without debugger ones."
+        )
+    else:
+        try:
+            from smdebug import exceptions
+
+            # list of exceptions debugger wants training toolkit to catch and log
+            exception_classes += [ex for ex in dir(exceptions) if isclass(getattr(exceptions, ex))]
+        except ImportError:
+            logger.info(
+                "sagemaker debugger is not installed, using exception types without debugger ones."
```

Comment:
Please also rename this. The word `ones` should be removed from the statement. May be rephrase this to 
`Exceptions not imported for SageMaker Debugger as it is not installed`